nixos/modules/services/hardware/pcscd.nix

```diff
@@ -16,17 +16,14 @@ let
 
 in
 {
-
-  ###### interface
```

Comment:
Could you explain why you delete these two comments?

I cannot see the benefits of doing so and it adds some "noise" to `git blame`.

Reply:
> Could you explain why you delete these two comments?

They are from the past. No new modules have them and they don't add any value.